specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/ServerConnectionPolicies.json

```diff
@@ -0,0 +1,269 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Lists a servers connection policies": {
+            "$ref": "./examples/ServerConnectionPoliciesList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerConnectionPolicyProperties": {
+      "description": "The properties of a server connection policy.",
+      "required": [
+        "connectionType"
+      ],
+      "type": "object",
+      "properties": {
+        "connectionType": {
+          "description": "The server connection type.",
+          "enum": [
+            "Default",
+            "Redirect",
+            "Proxy"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ServerConnectionType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "ServerConnectionPolicy": {
+      "description": "A server connection policy",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "description": "Resource location.",
+          "type": "string",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Metadata used for the Azure portal experience.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/ServerConnectionPolicyProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ServerConnectionPolicyListResult": {
+      "description": "A list of server connection policy objects.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServerConnectionPolicy"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ServerNameParameter": {
+      "name": "serverName",
+      "in": "path",
+      "description": "The name of the server.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
```

Comment:
I'm confused about why "x-ms-parameter-location": "method"? Isn't it coming from the URI, as is usual, not the method?